js/privatebin.js

```diff
@@ -3666,6 +3667,30 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             newDoc.close();
         }
 
+        /**
+         * download text
+         *
+         * @name   TopNav.downloadText
+         * @private
+         * @function
+         */
+        function downloadText()
+        {
+	    var filename='paste.txt';
```

Comment:
Tabulation

Reply:
Also, maybe we can set here paste id as filename?

Reply:
fixed in 853a4f386fa9a870cb2aa3616d43b61d0f10ec5f